styles/globals.css

```diff
@@ -44,3 +44,12 @@ a {
     background: black;
   }
 } */
+
+.mapboxgl-popup-content {
+  border-radius: 0;
+}
+
+.mapboxgl-popup-close-button {
+  font-size: x-large;
+  padding-right: 6px;
+}
```

Comment:
I'm sure there's a better place for this was wasn't getting much luck in a `GlobeEntry.module.css`. Appreciate this isn't ideal but happy to revisit.